### PR TITLE
Merge revision to develop

### PR DESCRIPTION
**Pull request by fabienrohrer**
Conflicts:

- [x] delete docs/guide/using-pycharm-with-webots.md (replaced by using-your-ide.md)
- [x] delete docs/guide/using-visual-studio-with-webots.md (replaced by using-your-ide.md)
- [x] resources/templates/controllers/c.vcxproj => Use the develop version, but applied unix2dos.
- [x] resources/templates/controllers/cpp.vcxproj => Use the develop version, but applied unix2dos.

Update:

- [x] Update `using-your.ide.md`: Fix lib paths.

### CODE REVIEW

**Comment by fabienrohrer** [comment]
@omichel Could you check I merged correctly modifications on *.vcxproj ?
@DavidMansolino Could you check I merged correctly modification on the lib path?

**Comment by fabienrohrer** [comment]
Build: https://travis-ci.com/cyberbotics/webots/builds/135302774

**Comment by omichel** [comment]
I checked the modifications on `*.vcxproj` and they look correct to me.

File: docs/guide/using-your-ide.md [hunk 1]
```diff
@@ -0,0 +1,384 @@
...
+
+%figure "Addition of the Webots controller library"
+
+![PyCharm Add Library](images/pycharm_add_lib.thumbnail.jpg)
+
+%end
+
+The Webots Python API depends on the Webots CPP API, therefore, the path need to be modifed to include the Webots `lib` directory.
+This can be done from the `Run` / `Edit Configurations` menu.
+In the `Run Configurations` windows, press the `+` button and then select `Python`, then set the `Script path` to point to your python file and in the `Environment variables` define the path variable (i.e. `PATH` on Windows, `LD_LIBRARY_PATH` on Linux or `DYLD_LIBRARY_PATH` on macOS) to point to `WEBOTS_HOME/lib/controller` (or `WEBOTS_HOME\msys64\mingw64\bin` on Windows).
```
Comment (omichel) [inline docs/guide/using-your-ide.md]:
```suggestion
In the `Run Configurations` windows, press the `+` button and then select `Python`, then set the `Script path` to point to your python file and in the `Environment variables` define the path variable (i.e. `PATH` on Windows, `LD_LIBRARY_PATH` on Linux or `DYLD_LIBRARY_PATH` on macOS) to point to `WEBOTS_HOME/lib/controller`.
```

**Pull request merged**